src/cmd/create.rs

```diff
@@ -12,13 +12,22 @@ use std::{
     path::{Path, PathBuf},
 };
 
+use strum_macros::EnumString;
+
 #[derive(Debug, StructOpt)]
 /// Create a new wallet
 pub enum Cmd {
     Basic(Basic),
     Sharded(Sharded),
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, EnumString)]
```

Comment:
Why do you need Copy and Clone?

Reply:
> This looks great! Just a few minor hygiene comments.
> 
> I like the use of strum, I hadn't seen that yet. It adds a little bloat that could be avoided with simple FromStr and Display implementations, but I think that's ok

@lthiery  suggested it, but I just found `--format` that does it more manually. I'll go either way. 

I needed Copy and Clone to do some if compares that I removed. not sure if I still do. I'll check.

Reply:
I can see Clone, but you hardly ever need to define Copy.

Reply:
And here's an example of a `Zone` enum which doesn't need all of heck to do it's work:

```rust
#[derive(Debug, Clone, PartialEq)]
pub enum Zone {
    Data,
    Config,
}

impl FromStr for Zone {
    type Err = Error;

    fn from_str(s: &str) -> Result<Self> {
        match s.to_lowercase().as_str() {
            "data" => Ok(Self::Data),
            "config" => Ok(Self::Config),
            _ => Err(Error::invalid_address()),
        }
    }
}

impl fmt::Display for Zone {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        match self {
            Self::Config => f.write_str("config"),
            Self::Data => f.write_str("data"),
        }
    }
}
```

Reply:
Ahh, I needed Copy, and Clone because I wasn't borrowing correctly, because I'm new to Rust. fixed. No more Copy or Clone and now taking immutable refs where I need to.